src/lib/masterSpan/index.js

```diff
@@ -0,0 +1,159 @@
+const crypto = require('crypto')
+const TraceParent = require('traceparent')
+const Catbox = require('@hapi/catbox')
+const CatboxMemory = require('@hapi/catbox-memory')
+const deserializeError = require('deserialize-error')
+const { merge } = require('lodash')
+const Logger = require('@mojaloop/central-services-logger')
+const { EventStatusType } = require('@mojaloop/event-sdk')
+const { tracer } = require('../tracer')
+const Config = require('../config')
+const partition = 'endpoint-cache'
+const clientOptions = { partition }
+let client
+
+const sleep = (ms) => {
```

Comment:
You might want to add this promise based sleep function [here](https://github.com/mojaloop/central-services-shared/blob/master/src/util/time.js). Please, do not overwrite the existing method, as it is used in unit testing without await.

Reply:
removed the need for sleep